src/modules/graphics/graphics.c

```diff
@@ -26,6 +26,11 @@ const char** os_vk_get_instance_extensions(uint32_t* count);
 #define MAX_SHADER_RESOURCES 32
 #define MATERIALS_PER_BLOCK 1024
 
+// 1GB is a very specific number because the gpu.h limits are always at least this large.
+// If this constant ever goes up, we will need to fetch the gpu limit and compare.
+#define MAX_BUFFER_SIZE (MAX_FRAME_MEMORY)
+#define MAX_BUFFER_SIZE_STR "1 GiB"
```

Comment:
I regret to report that the IEC officially defines 1 GB as 1000*1000*1000 and 1 "Gibibyte", a word no one ever uses, as 1024*1024*1024. I don't agree with this but error messages should probably be exact

Reply:
Hm, I'd almost rather do `1073741824`